core/Tracker/IgnoreCookie.php

```diff
@@ -61,7 +61,7 @@ public static function setIgnoreCookie()
             $ignoreCookie->delete();
         } else {
             $ignoreCookie->set('ignore', '*');
-            $ignoreCookie->save();
+            $ignoreCookie->save('None');
```

Comment:
should we move this for now into this PR that handles the opt out? https://github.com/matomo-org/matomo/pull/15184/files